Make target "install" is broken

`make install` fails with the following error:

```
./bin/python3 setup.py development
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'development'
Makefile:17: recipe for target 'install' failed
make: *** [install] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,7 @@ help:
 
 install:
 	npm install
-	$(VIRTUAL_ENV)/bin/python3 setup.py development
+	$(VIRTUAL_ENV)/bin/pip install -r requirements/dev.txt
 
 lint:
 	. $(VIRTUAL_ENV)/bin/activate && node_modules/.bin/polylint

